content/en/scale-agent/release-notes/agent-plugin/agent-plug-v-0-10-57.md

```diff
@@ -0,0 +1,8 @@
+---
+title: v0.10.57 Armory Agent Clouddriver Plugin (2022-10-25)
+toc_hide: true
+version: 00.10.57
+
+---
+
+Fix to cache namespaces independent of the `kubesvc.runtime.defaults.onlySpinnakerManaged` property value. This change allows the agent to recieve all Clouddriver accounts, mapped by namespace.
```

Comment:
Could we change this line :
`This change allows the agent to recieve all Clouddriver accounts, mapped by namespace.
`by:
`This change allows agent plugin to retrive all connected accounts by GET /credentials endpoint.`